### PR TITLE
Always highlight pre code

### PR DESCRIPTION
**Pull request by raphink**
CommonMarker doesn't place the "highlight" class on `<pre>` tags,
which fails to highlight them.

I can't think of a case where `<pre><code></code></pre>` shouldn't be
highlighted, and in such cases, you could always manually add the
`nohighlight` class, which is honored by highlightjs.

### CODE REVIEW

File: lib/showoff.rb [hunk 1]
```diff
@@ -858,7 +858,6 @@ def update_commandline_code(slide)
           lines = out.split("\n")
           if lines.first.strip[0, 3] == '@@@'
             lang = lines.shift.gsub('@@@', '').strip
-            pre.set_attribute('class', 'highlight')
```
Comment (binford2k) [inline lib/showoff.rb:861]: I'm confused as to what CommonMarker does differently that makes this not work. The rest of the code can only work if it's got nested `<pre><code>` blocks anyway. I'll poke a little and see what's being generated.

Comment (raphink) [reply lib/showoff.rb:861]: CommonMarker generates `<pre><code>`, while showoff generates `<pre class="highlight"><code>`. My suggestion is to remove the highlight class from `<pre>` in the CSS so it works with CommonMarker-generated HTML, and if we do that, then there's no need to add the highlight class anymore.

Comment (binford2k) [reply lib/showoff.rb:861]: The markdown renderer only generates `<pre><code>`. Showoff adds the `highlight` class.

I just tested it with the example presentation, and it properly highlights, whether using redcarpet or commonmarker. Do you have an example presentation that displays this behaviour?

Comment (raphink) [reply lib/showoff.rb:861]: The example doesn't work for me. Steps:

Replace

```
@@@javascript
the code
```

 with

````
```javascript
the code
```
````

 in example/one/slidesB.md and add `"markdown": "commonmarker",` to example/showoff.json. Result on http://localhost:9090/#8:

screenshot from 2017-01-16 12-46-12

**Comment by binford2k** [comment]
Thanks for finding and identifying the problem, @raphink! I corrected this in another way, so that only code blocks that are tagged with a language will be highlighted. This way, there aren't unexpected results with existing presentations. I think that I want to retain the behavior of requiring the language to be set explicitly for highlighting to occur. Fewer surprises that way :)

**Pull request closed without merging**